proxy-robots-module/src/main/java/orwell/proxy/robot/IPWebcam.java

```diff
@@ -0,0 +1,53 @@
+package orwell.proxy.robot;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import orwell.proxy.config.IConfigCamera;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class IPWebcam implements ICamera {
+    private final static Logger logback = LoggerFactory.getLogger(IPWebcam.class);
+    private final static String DUMMY_URL = "http://goo.gl/y1QTB7";
+    private final URL url;
+
+    public IPWebcam(final URL url) {
+        assert(null != url);
+        this.url = url;
+    }
+
+    public IPWebcam(final IConfigCamera configCamera) throws MalformedURLException {
+        assert(null != configCamera);
+        try {
+            if (null == configCamera.getResourcePath()) {
```

Comment:
Would it be possible to have getResourcePath return an empty string to avoid testing for null ?